GSoC/2016/README.md

```diff
@@ -0,0 +1,29 @@
+# Note for 2016 GSoC application
+
+It's my first time in the GSoC application. Due to my lack of the process, I missed the deadline of updating my enrollment letter, which finally caused the eligibility.
+
+This proposal was not selected as one of the GSoC project as the enrollment letter was not in the correct form. I didn't know this massage until I checked the GSoC website manually before the last day which was too late according to the rules. So unfortunately, this project will not be eligible.
+
+However, the project will go on, if anyone have interests in this idea, please check [QuCmp.jl](https://github.com/JuliaQuantum/QuCmp) in JuliaQuantum's repo.
+
+To prevent the same accident from happening again and to help people in region without Google services to get involved with all those amazing ideas, I'll leave some advise here for other students from where google is not available :
+
+### For students
+
+- If you have trouble in connecting GSoC's official website, (eg. the browser gives you 404 error), please ask JuliaQuantum members for help by email or gitter.
+- check your application status after your application regularly
+- if you do not use gmail very often or you have trouble in using gmail, please set your gmail automatically forward the emails to your commonly used email
+- Besides your proposal, check the enrollment letter, it has to be your university's official certificate in English, which could take some time in the preparation.
+
+### For mentors
+
+Since JuliaQuantum mentors may not originally come from its present members, and it's possible to invite people as your mentor, some mentors from China could be the first time in participating. So here are some suggestions for mentors from places that may have trouble in using Google services.
+
+- You may have trouble in reading through the mentor's guide and rules in GSoC's official website. If so, please feel free to ask in JuliaQuantum's email list or gitter channel for help.
+
+
+### Possible tools if you want to fix the connection
+
+[Lantern](https://github.com/getlantern/lantern), [shadowsock](https://github.com/ziggear/shadowsocks), and VPN services.
```

Comment:
[Lantern](https://github.com/getlantern/lantern), [shadowsock](https://github.com/ziggear/shadowsocks), and VPN services. Use at your own risk.